### ISSUE TITLE
Analyst can see what status the Applicant's dashboard shows

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#1358]
As an analyst
I want to know the status that is visible to the applicant
So that I don't mistakenly change the status withouht knowing the effect it will have on the status that the applicant will see

**Description:**
This ticket is about adding the 'External Status' to the header in the analyst portal.

# Acceptance Criteria:

**Screen:**

![image.png](https://images.zenhubusercontent.com/61ae5dab07758299775d9d78/d6be5358-924e-457b-a221-1cb8552fbee8)
[**Wireframe**](https://www.figma.com/file/nhOquOVmWD1P3BxEY0KIh5/CCBC-Intake?node-id=10767-84927&t=M9oHwsktSJI9w0t6-4)

**Page Behaviours**

- [x] The status that is visible to the applicant is also visible in the Analyst portal header.  
- [x] The applicant status shows as a READ ONLY pill (use pill colours already associated with analyst portal)
- [ ] The applicant status <s>changes when other functions on the analyst portal change the status for the applicant and </s>stays synchronized with what the applicant sees on their dashboard
- [x] The analyst status label reads 'Internal Status'
- [x] The other 'Status' label changes to 'External Status'

## Implementation Details
- **DB:**
  - Will need to create a computed column to query the status' with the restrictiveness of an applicant
- **Middleware:**
  - None
- **Frontend:**
  - Just need to rework that header area and reuse the pill
- **Tests:**
  - Update header tests to check for both pill statuses
- **Documentation:**
  - No update is required, does not change anything that requires an update of the project documentation

**Comment by elliottSlee** [comment]
@MarsRomer we need to include the ability to close applications at this point. Here are some notes from state machine conversations:
> analyst marks it closed before sending rejection letter. Applicant still sees in review, only when the letter has been sent that they see "rejected"

**Comment by MarsRomer** [comment]
Good point about the 'closed'.  I would be ok with not including that in this ticket and creating a new one for the following sprint, but we can discuss at stand up today
